docs/Help.md

```diff
@@ -68,6 +68,8 @@ Chatterino only makes a connection to the necessary APIs to fetch these emotes,
 Donate €10 to fourtf [here](https://streamelements.com/fourtf/tip) (can be done in multiple donations). You should get your badge automatically after about 5 minutes. Make sure to restart Chatterino if you can't see the badge.  
 If you still can't see it, ask fourtf on [Chatterino Discord](https://discord.gg/qq7DDxjste) for assistance.
 
+### How do I disable the supporter badge?
+login to the [manage page] (https://manage.chatterino.com/) and click on disable your Chatterino badge
```

Comment:
- New line at the end was missing.
 - Fix up grammar.
 - Stray space in markdown link syntax.

```suggestion
### How do I disable the supporter badge?
Log in to the [badge manager](https://manage.chatterino.com/) and click on *Disable My Chatterino Badge*.

```